src/Controller/CRUDController.php

```diff
@@ -1563,8 +1563,8 @@ private function setFormTheme(FormView $formView, ?array $theme = null): void
 
     private function handleXmlHttpRequestErrorResponse(Request $request, FormInterface $form): ?JsonResponse
     {
-        if (!\in_array('application/json', $request->getAcceptableContentTypes(), true)) {
-            @trigger_error('In next major version response will return 406 NOT ACCEPTABLE without `Accept: application/json`', E_USER_DEPRECATED);
+        if (0 === \count(array_intersect(['application/json', '*/*'], $request->getAcceptableContentTypes()))) {
```

Comment:
Since we don't need the resulting count, we could use `empty()` instead `0 === count()`.